### ISSUE TITLE
common feature characteristics issue

### ISSUE THREAD

**Issue by JimmyGitHubFr** [dicengine/dice#205]
Hello, 

Could you tell me if this picture represents well the weakness DICe has concerning thefeature identification in stereo mode please? I have tried applying different filters but I often get this issue concerning the identification. 

![fm_projective_trans_0](https://user-images.githubusercontent.com/81685835/116243133-e4ad5680-a766-11eb-954c-a2c1ddfa4075.png)

If I'm correct you are planning to work on this issue after the summer? Do you think resolving this issue requires a lot of ressources, people and files to modify? If "DICe_feature.cpp" is the only file to modify I can maybe try to upgrade it?

Best regards,

Jimmy

**Comment by dicengine** [comment]
Yes, this is one of the issues we're trying to put some effort behind since it impacts whether a correlation can get off to a good initial guess or not. It seems to me like there are enough features in the image for the feature matching to work. The main feature matching code is in the match_features() function of DICe_Feature.h/cpp. Feel free to experiment with what's in there and let me know if you have any success.

Also, just linking in the other issue where this discussion first started: https://github.com/dicengine/dice/issues/201

**Comment by dicengine** [comment]
By the way, you might try the latest release v3.0-beta.2. It has the most up to date cross-correlation initializer.
